product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Security"
+---
+
+EDB's cloud offering runs in your own cloud account, isolates your data from other users, and gives you control over our access to it. The key security features are:
+- **Data Isolation:** Clusters are installed and managed in your cloud environment. Complete segregation of your data is ensured: your data is never stored in a shared space, and compromise of another EDB customer's system does not put your system at risk.
+- **Granular Access Control:** You can use _Single Sign On_ and define your own sets of _Roles_ and _Role Based Access Control_ policies to manage your individual cloud environments.
```

Comment:
Two things here:

1. The use of italics seems a bit odd.
2. Can we link to Administering Your Account > Managing User Access here?

Reply:
See changes in #1692